docs/resources/database.md

```diff
@@ -1 +1,464 @@
-# :resources-database: Database
\ No newline at end of file
+# :resources-database: Database
+
+Database as a [Resource](/resources/) within DataOS is declared for creating a database within DataOS. It is treated as a first-class resource, encapsulating the fundamental unit for storing and organizing structured data using PostgreSQL. The classification of Databases as a resource type within the DataOS ecosystem allows for systematic management, provisioning, and utilization of databases in a standardized manner.
+
+<aside class=”callout”>
+
+ Database is a Workspace-level Resource type.
```

Comment:
A Database is a Workspace-level Resource-type, implying that its scope is limited to a particular Workspace.